plugins/services/src/js/components/forms/ContainerServiceFormAdvancedSection.tsx

```diff
@@ -268,6 +272,96 @@ class ContainerServiceFormAdvancedSection extends React.Component {
             <FieldError>{diskErrors}</FieldError>
           </FormGroup>
         </FormRow>
+        <Trans render="h2" className="short-bottom">
+          Limits
+        </Trans>
+        <Trans render="p">Limits settings for cpu and mem.</Trans>
+        <FormRow>
+          <FormGroup className="column-4" showError={Boolean(limitsErrors)}>
+            <FieldLabel className="text-no-transform">
+              <FormGroupHeading>
+                <FormGroupHeadingContent>
+                  <Trans render="span">CPUs</Trans>
+                </FormGroupHeadingContent>
+              </FormGroupHeading>
+            </FieldLabel>
+            <FieldInput
+              min="0"
+              name="limits.cpus"
+              step="0.01"
+              type="number"
+              value={
+                findNestedPropertyInObject(data, limitsPath + ".cpus.value") ||
+                (findNestedPropertyInObject(
+                  data,
+                  limitsPath + ".cpus.value"
+                ) === 0
+                  ? 0
+                  : "")
+              }
+              autoFocus={Boolean(limitsErrors)}
+              disabled={
+                findNestedPropertyInObject(
+                  data,
+                  limitsPath + ".cpus.unlimited"
+                ) === true
```

Comment:
can we have some types, so it becomes obvious that we need a strict equality check here?

Reply:
You mean prop types? Added something to the component.